ui/apps/platform/src/Containers/AccessControl/AuthProviders/ConfigurationFormFields.tsx

```diff
@@ -283,6 +283,29 @@ function ConfigurationFormFields({
                             />
                         </FormGroup>
                     </GridItem>
+                    <GridItem span={6} lg={6}>
+                        <FormGroup
+                            fieldId="config.disable_offline_access_scope"
+                            helperText={
+                                <span className="pf-u-font-size-sm">
+                                    Could be used if identity provider has limit on number of
+                                    offline tokens issued.
+                                </span>
+                            }
+                        >
+                            <Checkbox
+                                isChecked={config.disable_offline_access_scope as boolean}
+                                label="Disable 'offline_access' scope"
+                                id="config.disable_offline_access_scope"
+                                name="config.disable_offline_access_scope"
+                                aria-label="Disable 'offline_access' scope"
+                                onChange={(checked) => {
+                                    setFieldValue('config.disable_offline_access_scope', checked);
```

Comment:
Optional investigation to increase consistency:

Does `onChange={onChange}` work here, as on line 281 for the existing `Checkbox` element?

Reply:
It doesn't - mysteriously, it leads to field value being assigned to `[]` 🤔 . That's why I used `setFieldValue`

Reply:
Can you do a quick test to see if the other checkbox works correctly? Maybe it has the same problem.

Reply:
Do you mean `do_not_use_client_secret`? Yes, that one works properly and assigns value to boolean `true` and `false`